olive/systems/azureml/aml_evaluation_runner.py

```diff
@@ -42,12 +42,21 @@ def main(raw_args=None):
 
     # load model
     model_config = get_model_config(common_args)
-    model = ModelConfig.from_json(model_config).create_model()
-
     # load metric
     metric_config = json.load(open(metric_args.metric_config))
     metric = create_metric(metric_config, metric_args)
 
+    # HF model tokenizer will be loaded from HF model hub
```

Comment:
I think this is already done by the run config validator https://github.com/microsoft/Olive/blob/main/olive/workflows/run/config.py#LL67C1-L67C1? Or do we expect the metric data config to be changed somewhere else?

Reply:
This part logic in config.py is running when initiating Olive from config file. This does not apply to runtime when we got a new model to evaluate it.

Reply:
Once the metric is set, I don't think the `model_name` changes?  All models come from the same source model, so I think the `model_name` is the same. Otherwise we would have different data for different models.

Reply:
I am not fully familiar with how the new `data_config` behaves so you can ignore this XD

Reply:
It sounds reasonable. I was having an error before that the metric does not have such data for outputed onnx model but I can't remember the exact scenario. Let me test it here.

Reply:
Seems it works. Maybe my previous case happens when the tokenizer was loaded from local files.

Reply:
Thanks for checking!